common/src/main/resources/deployments/maven/redhat-artifactory-maven-settings.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: settings-mvn
+data:
+  settings.xml: |
+    <?xml version="1.0" encoding="UTF-8"?>
+    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+        <profiles>
+            <profile>
+                <id>redhat</id>
+                <repositories>
+                    <repository>
+                        <id>redhat</id>
+                        <name>Repository for Redhat dependencies</name>
+                        <url>https://maven.repository.redhat.com/ga/</url>
+                    </repository>
+                </repositories>
+                <pluginRepositories>
+                    <pluginRepository>
+                        <id>redhat</id>
+                        <name>Repository for Redhat dependencies</name>
+                        <url>https://maven.repository.redhat.com/ga/</url>
+                    </pluginRepository>
+                </pluginRepositories>
+            </profile>
+            <profile>
+                <id>artifactory</id>
+                <repositories>
+                    <repository>
+                        <id>quarkus-maven-repo-zip</id>
+                        <url>http://10.0.136.110:8040/artifactory/quarkus-maven-repo-zip</url>
```

Comment:
If the artifactory IP used to change, I can parametrise this if you like. (This would require more changes in the jenkins job too).

Reply:
I would personally like to see this parametrised.

The test suite is consumed by other parties, and it'd be neat to hide the IP.

Reply:
Sure. I've already updated the PR by adding a `ts.s2i.maven.remote.repository` new parameter. Then, we need to update the Jenkins job to add our Artifactory IP.